src/index.ts

```diff
@@ -22,19 +22,20 @@ import * as extend from 'extend';
 import * as is from 'is';
 import * as isHtml from 'is-html';
 import * as prop from 'propprop';
-import { DecorateRequestOptions, BodyResponseCallback } from '@google-cloud/common/build/src/util';
+import {DecorateRequestOptions, BodyResponseCallback} from '@google-cloud/common/build/src/util';
 import * as r from 'request';
 
-
 const PKG = require('../../package.json');
 
 /**
  * @typedef {object} ClientConfig
  * @property {string} [projectId] The project ID from the Google Developer's
  *     Console, e.g. 'grape-spaceship-123'. We will also check the environment
  *     variable `GCLOUD_PROJECT` for your project ID. If your app is running in
- *     an environment which supports {@link https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application Application Default Credentials},
- *     your project ID will be detected automatically.
+ *     an environment which supports {@link
+ * https://cloud.google.com/docs/authentication/production#providing_credentials_to_your_application
```

Comment:
In this and other places, the doc formatting ends up un-indented when it reaches the next line, where we normally have 4 spaces. Is that done by a formatting tool?

Reply:
Yes, this was `clang-format`